.travis.yml

```diff
@@ -1,88 +1,141 @@
 sudo: true
 language: rust
-rust:
-- nightly
 
 # all unlabeled jobs run at test. Only if all "test" jobs finish, will the publish job run
 stages:
   - test
-  - publish
+  # See notes below about why publish is not enabled
+  # - publish
+
+_install_unix_template: &_install_unix_template
+  rust: nightly
+  install:
+  # configuration borrowed from holochain-rust Makefile, showing how to install zmq for ubuntu trusty on travis
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+      echo "deb http://download.opensuse.org/repositories/network:/messaging:/zeromq:/release-stable/xUbuntu_14.04/ ./" >> /etc/apt/sources.list;
+      wget https://download.opensuse.org/repositories/network:/messaging:/zeromq:/release-stable/xUbuntu_14.04/Release.key -O- | sudo apt-key add;
+      sudo apt-get update -qq;
+      sudo apt-get install libzmq3-dev;
+    fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install zmq; fi
+  # install our own nodejs to get a reasonable version
+  - rm -rf ~/.nvm && git clone https://github.com/creationix/nvm.git ~/.nvm && (cd ~/.nvm && git checkout `git describe --abbrev=0 --tags`) && source ~/.nvm/nvm.sh && nvm install $TRAVIS_NODE_VERSION
+  # install our own yarn to make things work on osx
+  - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.10.1
+  - export PATH=$HOME/.yarn/bin:$PATH
+
+_install_windows_template: &_install_windows_template
+  os: windows
+  rust: nightly-2018-10-12-x86_64-pc-windows-msvc
+  install:
+  - choco install nodist
+  - choco upgrade yarn
+  - export PATH="/c/Program Files (x86)/Yarn/bin:/c/Program Files (x86)/Nodist/bin:$PATH"
+  - export NODE_PATH="/c/Program Files (x86)\Nodist\bin\node_modules;$NODE_PATH"
+  - export NODIST_PREFIX="/c/Program Files (x86)\Nodist"
+  - export NODIST_X64=1
+  - nodist add $TRAVIS_NODE_VERSION
+  - nodist $TRAVIS_NODE_VERSION
+  - node -e "console.log(process.argv[0], process.arch, process.versions)"
+  # bring in holochain-rust for prebuilt 4.1 zmq binary
+  - git clone https://github.com/holochain/holochain-rust.git
+  - export PATH=$PATH:/c/Users/travis/build/holochain/holochain-nodejs/holochain-rust/vendor/zmq/bin  
+  # deps for neon, found at https://guides.neon-bindings.com/getting-started/
+  - npm install --scripts-prepend-node-path=true --global --vs2015 --production windows-build-tools
+  - yarn config set python python2.7
+  - npm config set msvs_version 2015
+  - yarn config set msvs_version 2015 --global
 
 jobs:
   include:
-
   # PRs, pushes to master, and tags build on all target arches
   # if this is release tag, the resultant binary will be uploaded to github
-  - name: "Linux - Node 8"
+  - <<: *_install_unix_template
+    name: "Linux - Node 8"
     os: linux
     env:
     - TRAVIS_NODE_VERSION="8"
     - SKIP_DEPLOY=0
-    if: tag =~ /^v\d+\.\d+\.\d+/ OR branch = master OR type = pull_request
-  - name: "OSX - Node 8"
+    if: tag =~ /^v\d+\.\d+\.\d+/ OR branch = add-windows-build OR type = pull_request
```

Comment:
I'll change add-windows-build back to master before merging... Just set it this way to get Travis running